src/act/node/canister_method/query_method.rs

```diff
@@ -59,15 +65,17 @@ impl Deref for QueryMethod {
 
 impl Declare<Context> for QueryMethod {
     fn to_declaration(&self, context: &Context, _: String) -> Option<Declaration> {
+        let user_defined_name = &self.name;
         let function_declaration = self.generate_function_body(context);
         let macro_args = if context.cdk_name == "kybra" {
             self.generate_kybra_macro_args()
         } else {
             self.generate_not_kybra_macro_args()
         };
+
         Some(quote! {
             #[ic_cdk_macros::query(#macro_args)]
```

Comment:
Don't we have to rename this here? Otherwise the Wasm import will be incorrect right?

Reply:
It's renamed inside of `#macro_args`